backend/src/config.ts

```diff
@@ -5,6 +5,7 @@ type Config = {
     redis: RedisConfig;
```

Comment:
Why are we still keeping this?

Reply:
Only feed cache is replaced. Redis is still used as a semi-persistent in-memory cache for other things.